packages/cli/src/constants.ts

```diff
@@ -90,6 +90,7 @@ export const LICENSE_FEATURES = {
 	PROJECT_ROLE_ADMIN: 'feat:projectRole:admin',
 	PROJECT_ROLE_EDITOR: 'feat:projectRole:editor',
 	PROJECT_ROLE_VIEWER: 'feat:projectRole:viewer',
+	COMMUNITY_NODES_CUSTOM_REGISTRY: 'feat:communityNodes:customRegistry',
```

Comment:
I see it's not in the [license server setup](https://github.com/search?q=repo%3An8n-io%2Flicense-management-setup%20communityNodes&type=code) repo yet - we should include it there by the time we release this.

Reply:
yeah. was planning to update the product yaml files after checking with David or Cornelius.

Reply:
https://github.com/n8n-io/license-management-setup/pull/60